kratos/tests/math_utils/test_math_utils.cpp

```diff
@@ -595,8 +595,10 @@ namespace Kratos
             array_1d<double, 3> b = ZeroVector(3);
             b[0] = 1.0;
 
-            const array_1d<double, 3>  c = MathUtils<double>::CrossProduct(a, b);
-            const array_1d<double, 3>  d = MathUtils<double>::UnitCrossProduct(a, b);
+            array_1d<double, 3>  c, d;
+
+            MathUtils<double>::CrossProduct(c, b, a);
```

Comment:
I assumed that for CrossProduct the values were inverted as well... Is that right?

Reply:
yes, that's the reason for the deprecation